### ISSUE TITLE
When resizing a comment textbox, the next click to make a new comment is offset

### ISSUE THREAD

**Issue by nimsarr** [CodeGrade/bottlenose#214]
In the manual grader:
- Create a comment
- Resize the textbox
- Optionally type in the textbox
- Create another comment
- That last click on the sidebar to make the second comment seems to be offset by the vertical resize distance of the previous comment's textbox

**Comment by blerner** [comment]
Yeah, this is an unfortunate glitch/limitation in CodeMirror, which I don't think I have the ability to fix at the moment.  If you click elsewhere between resizing the textbox and creating a new comment, and in particular if you scroll at all, then the comments should show up just fine.
